Sources/Core/ParameterDataSource.swift

```diff
@@ -10,6 +10,14 @@ import MetalPerformanceShaders
 @available(iOS 11.0, *)
 public class ConvolutionDataSource: NSObject, MPSCNNConvolutionDataSource {
 
+    public func copy(with zone: NSZone? = nil) -> Any {
```

Comment:
Is this change also related to the new Xcode?

Reply:
To iOS 12 I guess, but yes